### PR TITLE
Strong Passwords feature for new  Developer Account created form the Admin Po…

### PR DESCRIPTION
**Pull request by HemaHG**
**What this PR does / why we need it**:

Enable strong passwords feature for  new Developer Account created from the admin Portal via `**/buyers/accounts/new`**

**Which issue(s) this PR fixes** 
https://issues.redhat.com/browse/THREESCALE-1747

Screenshot from 2020-06-08 11-41-48

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -528,7 +528,7 @@ def cas?
     end
 
     def machine?
-      minimal? || api? || created_by_provider? || open_id? || cas? || oauth2?
+      minimal? || api? || open_id? || cas? || oauth2?
```
Comment (macejmic) [inline app/models/user.rb:531]: @HemaHG it needs a description, why this change is being made.. i don't get it. maybe add it to a PR description. thank you

Comment (Martouta) [reply app/models/user.rb:531]: @HemaHG If this change is intended, it requires a test because it is not obvious at all.

**Comment by thomasmaas** [comment]
@HemaHG has this pr been abandoned, should it be closed?

**Comment by HemaHG** [comment]
The issue https://issues.redhat.com/browse/THREESCALE-5416 is still under discussion, I will close the PR
